assignment/checkpoints/terminal.txt

```diff
@@ -1,21 +1,22 @@
 # Type your responses on a new line
 
 # 1 - What is the command for listing the files in the current folder (directory)?
-
+    ls
 # 2 - What is the command for changing directories?
-
+    cd
 # 3 - What is the command for going "up" a folder (to a parent directory)?
-
+    ..
 # 4 - What is a quick way to create a new folder?
-
+    mkdir brand-new
 # 5 - What output would you expect when you run the command 'ls' in an empty folder? What about a folder containing files?
-
+    For the first question, nothing would pop up. On the second, I would get -bash: cd: new-file.js: Not a directory.
```

Comment:
Take another look at the second part to Question #5. Not sure this answer makes much sense.